app/components/work_packages/activities_tab/journals/item_component/details.sass

```diff
@@ -0,0 +1,38 @@
+.work-packages-activities-tab-journals-item-component-details
+    .journal-details-container
+        margin-left: 19px
+        min-height: 20px
+    .journal-details-container--empty--last--asc
+        display: none!important
+    .journal-details-header-container
+        margin-left: -14px
+    .timeline-icon
+        background-color: var(--bgColor-muted)
+        border-radius: 50%
+        width: 28px
+        height: 28px
+        text-align: center
+        padding-top: 3px
+        margin-top: -2px
+    .empty-line
+        margin-top: 0px!important
+        margin-bottom: 0px!important
+    .journal-detail-stem-line
+        position: relative
+        width: 20px
+    .journal-detail-stem-line::before
+        content: ""
+        position: absolute
+        top: 10px
+        left: 0
+        width: 100%
+        height: var(--borderWidth-thin, 1px)
+        background-color: var(--borderColor-default)
+        transform: translateY(-50%)
+    .journal-detail-description
+        // quick hack to adapt the current detail rendering to desired primerised design
+        i
```

Comment:
Avoid overriding the styles like this, it will not be maintainable. If you need help figuring out the expected behavior, reach out to @HDinger or @bsatarnejad

Reply:
Noted. I did this because I was told to not touch the rendering of the journal detail descriptions. So I don
t have control over the HTML rendered in these lines. I will ask Henriette how to better handle this